app.js

```diff
@@ -34,3 +34,52 @@ var stuff = [
 //   // console.log(stuff[i].name);
 //   // console.log(stuff[i].src);
 // }
+
+function findRandom(){
+  var tempArray = [];
+  while(tempArray.length < 3){
+    var rand = stuff[Math.floor(Math.random() * stuff.length)].src;
+    if(tempArray.indexOf(rand) === -1){
+      tempArray.push(rand);
+    }
+    // console.log(tempArray);
+  }
+  randomNumArray = tempArray;
+}
+findRandom();
+console.log(randomNumArray);
```

Comment:
Where could you put this log statement to make it more useful? Does it fun every time the function runs?